src/Wheniwork.php

```diff
@@ -12,172 +12,205 @@
  * @author Daniel Olfelt <daniel@thisclicks.com>
  * @version 0.1
  */
-class Wheniwork {
+class Wheniwork
+{
+    /**
+     * Library Version
+     */
+    const VERSION = '0.1';
 
-    const METHOD_GET = 'get';
-    const METHOD_POST = 'post';
-    const METHOD_PUT = 'put';
-    const METHOD_PATCH = 'patch';
+    /**
+     * HTTP Methods
+     */
+    const METHOD_GET    = 'get';
+    const METHOD_POST   = 'post';
+    const METHOD_PUT    = 'put';
+    const METHOD_PATCH  = 'patch';
     const METHOD_DELETE = 'delete';
 
-    private $api_key;
     private $api_token;
     private $api_endpoint = 'https://api.wheniwork.com/2';
-    private $api_headers = [];
-    private $verify_ssl   = false;
+    private $api_headers  = [];
+    private $verify_ssl   = FALSE;
 
     /**
      * Create a new instance
+     *
      * @param string $api_token The user WhenIWork API token
      * @param array $options Allows you to set the `headers` and the `endpoint`
      */
-    function __construct($api_token = null, $options = [])
+    function __construct($api_token = NULL, $options = [])
```

Comment:
[PSR-2.5](http://www.php-fig.org/psr/psr-2/) says we MUST use lowercase for `true`, `false`, and `null`. Sorry. :disappointed: